interoperability_report.py

```diff
@@ -822,120 +528,168 @@ def parser():
             required=True,
             type=str,
             metavar='publisher_name',
-            help='Publisher Shape Application')
+            help='Path to the Publisher shape_main application. \
+                If the executable is in the same folder as the script it should \
+                contain the "./". Example: ./rti_connext_dds-6.1.1_shape_main_linux')
         gen_opts.add_argument('-S', '--subscriber',
             default=None,
             required=True,
             type=str,
             metavar='subscriber_name',
-            help='Subscriber Shape Application')
+            help='Path to the Subscriber shape_main application. \
+                If the executable is in the same folder as the script it should \
+                contain the "./". Example: ./rti_connext_dds-6.1.1_shape_main_linux')
 
         optional = parser.add_argument_group(title='optional parameters')
         optional.add_argument('-v','--verbose',
             default=False,
             required=False,
             action='store_true',
-            help='Print more information to stdout.')
-
-        out_opts = parser.add_argument_group(title='output options')
-        out_opts.add_argument('-f', '--output-format',
-            default='junit',
+            help='Print debug information to stdout. It will track the \
+                interoperability_report execution and it will show the \
+                shape_main application output in case of error. \
+                By default is non selected and the console output \
+                will be the results of the tests.')
+
+        tests = parser.add_argument_group(title='Test Case and Test Suite')
+        tests.add_argument('-s', '--suite',
+            nargs='+',
+            default=['test_suite'],
             required=False,
+            metavar='test_suite_dictionary_file',
             type=str,
-            choices=['junit', 'csv', 'xlxs'],
-            help='Output format.')
+            help='Test Suite that is going to be tested. \
+                Test Suite is a file with a dictionary defined, it should \
+                be located on the same directory as interoperability_report. \
+                By default is test_suite. \
```

Comment:
It is not shown (at least I don't see it)

Reply:
I don't know if it is a good idea to use it because:
* with -t (enable test_case) automatically the default option will be printed on the console as None (and it should be All)
* with -o (output report name) automatically the default option will be printed on the console as None (and it should be <publisher>---<subscriber>-date.xml
And later in the code we are using if (<> is None)....; we should also change in the code that and replace it for 
* if enable_test is ALL...
* if output_name is 'publisher-subscriber-date.xml'...
I dont know if it will make the code more difficult to understand
Plus, None is recognized variable but All not

Reply:
okey

Reply:
done

Reply:
we need to add it to all parameters, following the same format that when it is printed by `argparse`, eg (Default: <default_value>)

Reply:
done